flag.go

```diff
@@ -46,40 +56,59 @@ type FlagLoader struct {
 	// that will used in passed into the flag for Usage.
 	FlagUsageFunc func(name string) string
 
+	// ErrorHandling matches flag.ErrorHandling value
+	//  By Default this is set to flag.ExitOnError for backwards compatibility.
+	//  Other values include flag.ContinueOnError and flag.PanicOnError
+	ErrorHandling ErrorHandling
+
 	// only exists for testing.  This is the raw flagset that is to parse
 	flagSet *flag.FlagSet
 }
 
+func (f *FlagLoader) Help() string {
+	out := new(bytes.Buffer)
+	f.flagSet.SetOutput(out)
+	f.flagSet.PrintDefaults()
+	return out.String()
+}
+
 // Load loads the source into the config defined by struct s
 func (f *FlagLoader) Load(s interface{}) error {
 	strct := structs.New(s)
 	structName := strct.Name()
 
-	flagSet := flag.NewFlagSet(structName, flag.ExitOnError)
+	var flagSet *flag.FlagSet
```

Comment:
I think instead of adding new function (Help()) the code you have added here is sufficient, since there is no way for us the call Help on error cases, letting std lib to handle it with flag.Usage is fine. On the other hand we already leaked our abstractions (put environment loader into flag loader) in other places, this looks OK for me.